tests/integration/api/v1/contents/[username]/get.test.js

```diff
@@ -859,6 +859,14 @@ describe('GET /api/v1/contents/[username]', () => {
       expect(uuidVersion(responseBody[0].owner_id)).toEqual(4);
       expect(uuidVersion(responseBody[1].owner_id)).toEqual(4);
       expect(responseBody[0].published_at > responseBody[1].published_at).toEqual(true);
+
+      const responseLinkHeader = parseLinkHeader(response.headers.get('Link'));
+      expect(responseLinkHeader.first.url).toStrictEqual(
```

Comment:
Não sei se a diferença de desempenho compensa a correção disso em todos os testes existentes, mas, pelo menos nos novos testes, e nos que eu mexi por algum motivo, eu venho usando `toBe` para as comparações de primitivos.

Então eu substituiria estes `toStrictEqual` daqui, e `toEqual` do outro teste, por `toBe`:

```suggestion
      expect(responseLinkHeader.first.url).toBe(
```

Reply:
Em outros projetos, eu costumo usar `.toBe()` e `toEqual()` conforme o que é mais adequado, então não tenho nada contra essa alteração.